fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -37,6 +37,9 @@
     private static final Logger LOGGER = LoggerFactory
             .getLogger(FedoraUserSecurityContext.class);
 
+    // Defined here because FedoraJcrTypes is in fcrepo-http-api, which would create a cyclic dependency
+    protected static final String JCR_CONTENT = "jcr:content";
```

Comment:
`FedoraJcrTypes` is in fcrepo-kernel-api, and if `jcr:content` were defined there, I'm not sure that I see how a cyclic dependency would be created.